cluster/addons/device-plugins/nvidia-gpu/daemonset.yaml

```diff
@@ -26,6 +26,8 @@ spec:
       - key: "nvidia.com/gpu"
         effect: "NoSchedule"
         operator: "Exists"
+      - key: "CriticalAddonsOnly"
+        operator: "Exists"
```

Comment:
I assume that device plugin should never be evicted. So, having no effect should be fine.

Reply:
Yes.